src/main/kotlin/org/rust/cargo/project/workspace/CargoWorkspace.kt

```diff
@@ -28,6 +28,7 @@ interface CargoWorkspace {
 
     val packages: Collection<Package>
     fun findPackage(name: String): Package? = packages.find { it.name == name }
+    fun findPackageByNormName(normName: String): Package? = packages.find { it.normName == normName }
```

Comment:
Maybe it's better to upgrade `findPackage` method (i.e. search package by name and norm name at the same time) instead of new rather specific method in API?

Reply:
:thinking: :thinking: :thinking: